doc(volume): max volume size for the file system (#831)
longhorn/website: content/docs/1.6.0/volumes-and-nodes/expansion.md

- The filesystem used in the Longhorn volume is one of the following:
    - ext4
    - xfs

> - The expanded size should be smaller than the maximum file size of the file system, e.g., 16TiB for the `ext4` file system.

Reviewer: ```suggestion
The expanded size must be less than the maximum file size allowed by the file system (for example, 16TiB for `ext4`).
```

Author: Modified. Thanks.
